evalml/automl/automl_algorithm/default_algorithm.py

```diff
@@ -257,12 +265,28 @@ def _create_fast_final(self):
             if estimator not in self._naive_estimators()
         ]
 
-        pipelines = [
-            self._make_split_pipeline(
-                estimator,
-            )
-            for estimator in estimators
-        ]
+        if is_time_series(self.problem_type):
```

Comment:
Theres a couple `is_time_series` scattered in this PR and I like how explicit it is (although it does make this code more verbose). An alternative would be to move `is_time_series` into `_make_split_pipeline`.